config/vars.go

```diff
@@ -32,6 +32,5 @@ var (
 	// SkipRelaySignatureCheck can be used to disable relay signature check
 	SkipRelaySignatureCheck = os.Getenv("SKIP_RELAY_SIGNATURE_CHECK") == "1"
 
-	GenesisTime = int64(common.GetEnvInt("GENESIS_TIMESTAMP", common.GenesisTimeMainnet))
```

Comment:
think we'll still want this env var for custom devnets

Reply:
if set, this should always be used, whatever flag was used.

Reply:
I add a commit for that. However at the end, `time.Now().UTC().UnixMilli()` is `int64` not `uint64`, so we still need a cast. IMO it would be better to consider all times as `int64`.

Reply:
int64 seems ok